### PR TITLE
[OP-NN] Fix for deprecated old style qupulse Sequencer test

### PR DESCRIPTION
**Pull request by QFer**
* Now testing to the calculated values of qupulse 0.4
* Also the tests were not testing the right ['sawtooth'] values. Adjusted.

### CODE REVIEW

File: src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py [hunk 1]
```diff
@@ -118,24 +118,17 @@ def test_qupulse_template_to_array_new_style_vs_old_style(self):
...
                                            to_single_waveform=set())
-
             (_, voltages_new, _) = render(loop, sampling_rate / 1e9)
-            self.assertTrue(len(voltages_old) == len(voltages_new))
-            self.assertTrue(np.min(voltages_old) == np.min(voltages_old))
-            self.assertTrue(np.max(voltages_old) == np.max(voltages_old))
+
+            # the value to compare to are calculated using qupulse 0.4 Sequencer class
+            self.assertTrue(1000001 == len(voltages_new['sawtooth']))
+            self.assertAlmostEqual(-0.75, np.min(voltages_new['sawtooth']), 12)
```
Comment (peendebak) [inline src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py:130]:
```suggestion
            self.assertAlmostEqual(-amplitude/2, np.min(voltages_new['sawtooth']), 12)
```

File: src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py [hunk 2]
```diff
@@ -118,24 +118,17 @@ def test_qupulse_template_to_array_new_style_vs_old_style(self):
...
+            self.assertAlmostEqual(0.75, np.max(voltages_new['sawtooth']), 12)
```
Comment (peendebak) [inline src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py:131]:
```suggestion
            self.assertAlmostEqual(amplitude/2, np.max(voltages_new['sawtooth']), 12)
```

File: src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py (hunk 1 above)
Comment (QFer) [reply src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py]: changed it

File: src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py (hunk 2 above)
Comment (QFer) [reply src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py]: changed it

**Pull request merged**